Makefile.toml

```diff
@@ -401,16 +401,30 @@ for m in ${GO_MODULES}; do
 done
 
 # For rust first-party source code
-if ! cargo fmt \
-  --manifest-path ${BUILDSYS_SOURCES_DIR}/Cargo.toml \
+if ! docker run --rm \
+   -u $(id -u):$(id -g) \
+   -e CARGO_HOME="/tmp/.cargo" \
+   -v "${CARGO_HOME}":/tmp/.cargo \
+   -v "${BUILDSYS_ROOT_DIR}/sources":/tmp/sources \
+   -v "${BUILDSYS_ROOT_DIR}/tools":/tmp/tools \
```

Comment:
```suggestion
```
Do we need `tools` mounted if we're just checking `sources`? I suspect it can be removed and the reverse is true for the next block. We might also want to make the same change to the `clippy` piece as well (in a separate commit).

Reply:
We need `sources` mounted in both blocks but `tools` is indeed only necessary when the `tools` manifest is checked. I'll go ahead and change that.

Reply:
Ah. It looks like a couple of `tools` are leveraging `parse-datetime` from `sources`.